README.md

```diff
@@ -100,137 +45,74 @@ After you're finished please remove all the comments and instructions!
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- Database
```

Comment:
- [ ] [OPITONAL] As a developer, it's important to not only write clean and well-organized code but also to document your work effectively. One key aspect of this is creating a detailed and informative README file. In particular, the Key Features & Future Features section of your README presents a great opportunity to showcase your creativity and vision for the project. I would highly recommend adding a compelling and descriptive feature to this section to grab the attention of recruiters or future clients. 😊